app/importers/unpaywall_publication_importer.rb

```diff
@@ -107,6 +107,6 @@ def update_open_access_location(open_access_location, unpaywall_json)
     end
 
     def build_new_oal(publication, url)
-      publication.open_access_locations.build(source: Source::UNPAYWALL, url: url)
+      publication.open_access_locations.build(source: Source::USER, url: url)
```

Comment:
This should stay as `Source::UNPAYWALL`, right?

Reply:
oh yes! thanks for catching this, totally mistakenly changes.

Reply:
fixed now.